app/experimenter/nimbus-ui/src/components/AppLayoutSidebarLaunched/index.test.tsx

```diff
@@ -79,7 +88,7 @@ describe("AppLayoutSidebarLaunched", () => {
         `${BASE_PATH}/my-special-slug`,
       );
 
-      screen.getByText("Experiment results not yet ready");
+      screen.getByText("Experiment analysis not ready yet");
```

Comment:
I changed this text to use the word "analysis" since "Results" sits right above this and this just felt nicer since the title on the page is "Analysis"? I can easily revert if preferred.